docs/developers.md

```diff
@@ -21,7 +21,9 @@ We use Make to build our programs. The simplest way to get started is:
 $ make bootstrap build
 ```
 
-NOTE: This will fail if not run from the path: `$GOPATH/src/k8s.io/helm`.
+NOTE: This will fail if not running from the path `$GOPATH/src/k8s.io/helm`. The
+directory `k8s.io` should not be a symlink or `build` couldn't find the relevant
```

Comment:
Consider using the correct matching tense to make this more clear i.e.
"The directory `k8s.io` should not be a symlink or `build` will not find..."

Reply:
fixed